### DISCUSSION TITLE
2008/01/04/%E9%95%BF%E9%93%97%E8%A7%A3%E6%A2%A6

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#203] [Announcements]
# 2008/01/04/%E9%95%BF%E9%93%97%E8%A7%A3%E6%A2%A6

中国俗语说日有所思，夜有所梦；弗洛伊德说梦是在满足潜意识下的欲望；巴普洛夫说梦是警戒点在普遍抑制的大脑皮层上的游走；我说梦是大脑再做内存碎片整理。

https://blog.qizhen.xyz/2008/01/04/%E9%95%BF%E9%93%97%E8%A7%A3%E6%A2%A6

**Comment by ruanqizhen** [comment]
之前博客下的留言：

xiaobai
2008年03月26日 19:36 编辑
梦是大脑神经活动的一种方式，等同于yy。
而yy有，希望是和希望不是2种。
以上两面性等同没说的废话。

回复
竹贤
2008年01月5日 06:35 编辑
其实，无论是弗洛伊德、巴普洛夫还是阮大侠，都认可梦是一种大脑皮层的思维活动。只是目前人们对为何有这种活动，以及如何活动，还没有完全认识而已。

回复
竹贤
2008年01月4日 20:37 编辑
一般说来，人多多少少是有一点惰性的，然而，意志又能够克服惰性；良好的生活习惯能够使自己远离惰性。

回复
Arthur
2008年01月4日 19:25 编辑
做梦是不是会降低睡眠质量的？好像有这种说法。

回复

**Comment by ruanqizhen** [comment]
之前博客下的留言：

LIVIA
2008年02月26日 18:45 编辑
好神奇~

回复
竹贤
2008年01月11日 08:15 编辑
看来“眼见为实”的说法也有局限性啊。

回复
鱼
2008年01月8日 13:17 编辑
是啊，很专业

回复
WL
2008年01月8日 04:23 编辑
有道理啊

回复
acca
2008年01月8日 00:17 编辑
突然发现这里的文章毛专业…
略作了解也不错~

回复
竹贤
2008年01月7日 21:39 编辑
确实如此。
